src/response/file.rs

```diff
@@ -233,8 +233,14 @@ impl Stream for FileStream {
                 let filled = data.filled();
 
                 if let Some(remaining) = this.remaining.checked_sub(filled.len()) {
+                    let data = Bytes::copy_from_slice(filled);
                     this.remaining = remaining;
-                    Poll::Ready(Some(Ok(Bytes::copy_from_slice(filled))))
+
+                    if remaining == 0 {
+                        this.buffer.fill(MaybeUninit::uninit());
```

Comment:
Nothing wrong with modifying `this.buffer` here as the borrow checker allows it and the data from the filled portion of the buffer is copied into the Bytes instance instantiated on 236.

Reply:
Also, I may be wrong about this but the bytes crate updated to guarantee stable memory address which should mean that `16kb` is not copied on `236` and `243`. Only struct wrapper is copied.